PWG/EMCAL/EMCALbase/AliAnalysisTaskEmcal.cxx

```diff
@@ -362,13 +367,21 @@ void AliAnalysisTaskEmcal::UserCreateOutputObjects()
     AliError("Analysis manager not found!");
   }  
 
+  if(!fUseInternalEventSelection) {
+    fAliEventCuts = new AliEventCuts(true);
+    // Do not perform trigger selection in the AliEvent cuts but let the task do this before
+    fAliEventCuts->OverrideAutomaticTriggerSelection(AliVEvent::kAny, true);
+  }
+
   if (!fCreateHisto)
     return;
 
   OpenFile(1);
   fOutput = new AliEmcalList();
   fOutput->SetUseScaling(fUsePtHardBinScaling);
   fOutput->SetOwner();
+  
+  if(fAliEventCuts) fOutput->Add(fAliEventCuts);
```

Comment:
Does this work? I had always added a TList of the AliEventCuts QA plots here.

Reply:
I use this in my task where I overwrote the IsEventSelected method and have no problems. Adding the event cuts to the output list does not create any troubles, also since it inherits from TList.